src/components/FadeTransitioner/FadeTransitioner.tsx

```diff
@@ -29,7 +28,7 @@ const childNodeStyles = {
  */
 function FadeTransitioner({ nodeKey = '', children }: Props) {
   return (
-    <TransitionGroup>
+    <TransitionGroup style={{ minHeight: '100vh' }}>
```

Comment:
since it's static, this can be defined above the component

Reply:
good call, done